tests/Language/PrinterTest.php

```diff
@@ -86,6 +86,18 @@ public function testCorrectlyPrintsOpsWithoutName(): void
 }
 ';
         self::assertEquals($expected, Printer::doPrint($mutationAstWithArtifacts));
+
+        $queryWithNonNullVariable = Parser::parse(
+            'query ($var: ID!) { a(arg: $var) { __typename } }
+'
+        );
+        $expected                 = 'query ($var: ID!) {
+  a(arg: $var) {
+    __typename
+  }
+}
+';
+        self::assertEquals($expected, Printer::doPrint($queryWithNonNullVariable));
```

Comment:
Could you move it to a separated test method without `@see it` annotation so that's clear it's an extra test not originating from `graphql-js`? It's to make sure it's not removed later when `PrinterTest.php` synced with `printer-test.ts`. Thanks.